utils/clientId.go

```diff
@@ -13,9 +13,9 @@ import (
 // GenerateClientIDFromRequest generates a client id from the provided request.
 func GenerateClientIDFromRequest(req *http.Request) string {
 	ipAddress := GetIPAddressFromRequest(req)
-	ipAddressComponents := strings.Split(ipAddress, ":")
+	ipAddressComponents := strings.Split(ipAddress, ".")
 	ipAddressComponents[len(ipAddressComponents)-1] = ""
-	clientID := strings.Join(ipAddressComponents, ":") + req.UserAgent()
+	clientID := strings.Join(ipAddressComponents, ".") + req.UserAgent()
```

Comment:
It probably makes sense just to simplify this, since the added complexity isn't needed to generate this identifier and relies on a specific IP version.  How about just:

```suggestion
  clientID := ipAddress + req.UserAgent()
```

And remove the rest?